docs/reference/releases/rev156_136.md

```diff
@@ -0,0 +1,50 @@
+# Revision 156/136
+<sub>Thursday, July 4, 2024</sub>
+
+Dear community,
+
+We are glad to report that we have just released a new updated version for Charmed ZooKeeper on the `3/stable` channel, upgrading its revision from 126 to 136. 
+The release of a new version was promoted by the need of backporting some features that 
+should provide increased robustness and resilience during operation as well as smaller workload upgrades and fixes. See the technical notes for further information.
+
+Please reach out should you have any question, comment, feedback or information. You can find us here in [Matrix](https://matrix.to/#/#charmhub-data-platform:ubuntu.com) or also on [Discourse](https://discourse.charmhub.io/).
+
+## Features
+
+* [DPE-3726] Workload upgrade to 3.8.4-ubuntu0
+* [DPE-4183] (backport) fix: only handle quorum removal on relation-departed
+* [DPE-4362] (backport) add alive check fix
+
+Canonical Data issues are now public on both [Jira](https://warthogs.atlassian.net/jira/software/c/projects/DPE/issues/) 
+and [GitHub](https://github.com/canonical/kafka-operator/issues) platforms.
+
+[GitHub Releases](https://github.com/canonical/kafka-operator/releases) provide a detailed list of bugfixes, PRs, and commits for each revision.
+
+## Inside the charms:
```

Comment:
The idea is the same as the previous release notes. Generally speaking - we don't want a colon to be a part of a heading and a list needs an introductory sentence or at the very least some text between the list and the previous heading.